guidelines/example_en-us.md

```diff
@@ -1,10 +1,12 @@
 # Example Style Guide English, USA (en-US)
 
-This style guide provides instruction for translating to the American English locale. This style guide is to be used in coordination with the [General Mozilla L10n Style Guide](general_L10n_SG.md)
+<!-- toc -->
+
+This style guide provides instruction for translating to the American English locale. This style guide is to be used in coordination with the [General Mozilla L10n Style Guide](mozilla_general/README.md)
```

Comment:
the link doesn't work for me, same issue at the end of the this doc when I click on it in "view".

Reply:
Yes, it's missing a .. at the beginning